docs/design/w3c/spec.md

```diff
@@ -0,0 +1,545 @@
+## W3C Verifiable Credentials Representation
+
+This section describes how Indy-styled AnonCreds credentials can be represented in the form of W3C Verifiable
+Credentials standard.
+
+### Credential
+
+This section describes how [W3C credential concepts](https://www.w3.org/TR/vc-data-model/#basic-concepts) are applied to
+AnonCreds W3C credential representation.
+
+Example of an AnonCreds W3C formatted credential which will be explained in details:
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://github.io/anoncreds-w3c/context.json"
+  ],
+  "type": [
+    "VerifiableCredential",
+    "AnonCredsCredential"
+  ],
+  "issuer": "did:sov:3avoBCqDMFHFaKUHug9s8W",
+  "issuanceDate": "2023-10-26T01:17:32Z",
+  "credentialSchema": {
+    "type": "AnonCredsDefinition",
+    "definition": "did:sov:3avoBCqDMFHFaKUHug9s8W:3:CL:13:default",
+    "schema": "did:sov:3avoBCqDMFHFaKUHug9s8W:2:basic_person:0.1.0",
+    "encoding": "auto"
+  },
+  "credentialSubject": {
+    "firstName": "Alice",
+    "lastName": "Jones",
+    "age": "18"
+  },
+  "proof": [
+    {
+      "type": "CLSignature2023",
+      "signature": "AAAgf9w5.....8Z_x3FqdwRHoWruiF0FlM"
+    },
+    {
+      "type": "Ed25519Signature2020",
+      "created": "2021-11-13T18:19:39Z",
+      "verificationMethod": "did:sov:3avoBCqDMFHFaKUHug9s8W#key-1",
+      "proofPurpose": "assertionMethod",
+      "proofValue": "z58DAdFfa9SkqZMVPxAQpic7ndSayn1PzZs6ZjWp1CktyGesjuTSwRdoWhAfGFCF5bppETSTojQCrfFPP2oumHKtz"
+    }
+  ]
+}
+```
+
+#### Context
+
+W3C [Context](https://www.w3.org/TR/vc-data-model/#contexts) section requires including of `@context` property to
+verifiable credential.
+
+The value of the `@context` property must be one or more resolvable [URI](https://www.w3.org/TR/vc-data-model/#dfn-uri)
+that result in machine-readable [JSON-LD](https://www.w3.org/TR/vc-data-model/#json-ld) information about the object
+format.
+
+The **context** definition used for AnonCreds W3C credentials representation can be found [here](./context.json).
+
+In the case of W3C AnonCreds credentials, the `@context` attribute includes an extra
+entry `https://github.io/anoncreds-w3c/context.json`
+which is required for the resolution of custom structure definitions and looks the following:
+
+```
+{
+  ...  
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://github.io/anoncreds-w3c/context.json"
+  ],
+  ...
+}
+```
+
+#### Identifiers
+
+W3C [Identifiers](https://www.w3.org/TR/vc-data-model/#identifiers) section defines an optional capability to assign
+some kind of identifier to the verifiable credential so that others can express statements about the same thing.
+
+In the case of W3C AnonCreds credentials, the `id` attribute is not recommended to set.
```

Comment:
For AnonCreds v1 — this work — this is the case.  In a future AnonCreds, optional metadata **could** be added to the AnonCreds data to identify an “id” attribute that could be used, but definitely not in this round, where we are not changing AnonCreds at all.